### DISCUSSION TITLE
Option to avoid hoisting in npm workspaces

### DISCUSSION THREAD

**Discussion by trusktr** [npm/feedback#575] [CLI] [answered]
`npm install` hoists dependencies to the root, and there is no option to not hoist (like with lerna). This means that some packages that the root package does not depend on are "installed" in the root package, which can be undesirable.

**Comment by MylesBorins** [comment, marked as answer]
There is an existing RFC to introduce a new reification mode called "isolated mode"

https://github.com/npm/rfcs/blob/main/accepted/0042-isolated-mode.md

This might be worth referencing if you want to make a similar proposal

**Reply by trusktr** [reply]
It seems like that will do the trick. Thanks!
